src/main/java/com/coolkids/coolKidsApp/services/EventServiceImpl.java

```diff
@@ -17,6 +17,10 @@ public class EventServiceImpl implements EventService {
 
     private final EventRepository eventRepository;
 
+    //This is in the Udemy course example (Video #193 @ 11:04) but not UserServiceImpl
+    //probably because of eventmapper.
```

Comment:
implement it and test it out to see that it works

Reply:
The current implementation works so I'll wait until I'm caught up with the rest of the app. If that's cool

Reply:
sounds good, let me know if I can help